runtime/parachains/src/hrmp/weights.rs

```diff
@@ -0,0 +1,114 @@
+		(0 as Weight)
+			// Standard Error: 19_000
+			.saturating_add((17_052_000 as Weight).saturating_mul(i as Weight))
+			// Standard Error: 19_000
+			.saturating_add((16_998_000 as Weight).saturating_mul(e as Weight))
+			.saturating_add(RocksDbWeight::get().reads(2 as Weight))
+			.saturating_add(RocksDbWeight::get().reads((2 as Weight).saturating_mul(i as Weight)))
+			.saturating_add(RocksDbWeight::get().reads((2 as Weight).saturating_mul(e as Weight)))
+			.saturating_add(RocksDbWeight::get().writes(4 as Weight))
+			.saturating_add(RocksDbWeight::get().writes((3 as Weight).saturating_mul(i as Weight)))
+			.saturating_add(RocksDbWeight::get().writes((3 as Weight).saturating_mul(e as Weight)))
+	}
+	// Storage: Configuration ActiveConfig (r:1 w:0)
+	// Storage: Hrmp HrmpOpenChannelRequestsList (r:1 w:0)
+	// Storage: Hrmp HrmpOpenChannelRequests (r:2 w:2)
+	// Storage: Paras ParaLifecycles (r:4 w:0)
+	// Storage: Hrmp HrmpIngressChannelsIndex (r:2 w:2)
+	// Storage: Hrmp HrmpEgressChannelsIndex (r:2 w:2)
+	// Storage: Hrmp HrmpOpenChannelRequestCount (r:2 w:2)
+	// Storage: Hrmp HrmpAcceptedChannelRequestCount (r:2 w:2)
+	// Storage: Hrmp HrmpChannels (r:0 w:2)
+	fn force_process_hrmp_open(c: u32) -> Weight {
+		(0 as Weight)
+			// Standard Error: 28_000
+			.saturating_add((37_628_000 as Weight).saturating_mul(c as Weight))
+			.saturating_add(RocksDbWeight::get().reads(2 as Weight))
+			.saturating_add(RocksDbWeight::get().reads((7 as Weight).saturating_mul(c as Weight)))
+			.saturating_add(RocksDbWeight::get().writes(1 as Weight))
+			.saturating_add(RocksDbWeight::get().writes((6 as Weight).saturating_mul(c as Weight)))
+	}
+	// Storage: Hrmp HrmpCloseChannelRequestsList (r:1 w:0)
+	// Storage: Hrmp HrmpChannels (r:2 w:2)
+	// Storage: Hrmp HrmpEgressChannelsIndex (r:2 w:2)
+	// Storage: Hrmp HrmpIngressChannelsIndex (r:2 w:2)
+	// Storage: Hrmp HrmpCloseChannelRequests (r:0 w:2)
+	// Storage: Hrmp HrmpChannelContents (r:0 w:2)
+	fn force_process_hrmp_close(c: u32) -> Weight {
+		(0 as Weight)
+			// Standard Error: 17_000
+			.saturating_add((21_825_000 as Weight).saturating_mul(c as Weight))
+			.saturating_add(RocksDbWeight::get().reads(1 as Weight))
+			.saturating_add(RocksDbWeight::get().reads((3 as Weight).saturating_mul(c as Weight)))
+			.saturating_add(RocksDbWeight::get().writes(1 as Weight))
+			.saturating_add(RocksDbWeight::get().writes((5 as Weight).saturating_mul(c as Weight)))
+	}
+	// Storage: Hrmp HrmpOpenChannelRequests (r:1 w:1)
+	// Storage: Hrmp HrmpOpenChannelRequestsList (r:1 w:1)
+	// Storage: Hrmp HrmpOpenChannelRequestCount (r:1 w:1)
+	fn hrmp_cancel_open_request(c: u32) -> Weight {
+		(35_031_000 as Weight)
+			// Standard Error: 0
+			.saturating_add((56_000 as Weight).saturating_mul(c as Weight))
+			.saturating_add(RocksDbWeight::get().reads(3 as Weight))
+			.saturating_add(RocksDbWeight::get().writes(3 as Weight))
+	}
+
+	fn clean_open_channel_requests(_: u32) -> Weight {
+		0
```

Comment:
Why is this 0?

Reply:
actually this file should be removed. We don't need a `impl WeightInfo for ()`.